README.md

```diff
@@ -82,7 +82,7 @@ The Makefile provides several targets:
 
   * The source code is periodically indexed: [Prometheus Core](http://godoc.org/github.com/prometheus/prometheus).
   * You will find a Travis CI configuration in `.travis.yml`.
-  * All of the core developers are accessible via the [Prometheus Developers Mailinglist](https://groups.google.com/forum/?fromgroups#!forum/prometheus-developers) and the `#prometheus` channel on `irc.freenode.net`.
+  * You can reach the Prometheus developers and users on various communication channels. For details, see the [Community page](https://prometheus.io/community).
```

Comment:
How about "See the community page for how to reach..."?
It's a little shorter and mentions the link earlier